<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StocksWindow</class>
 <widget class="QMainWindow" name="StocksWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stocks</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableViewStocks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>311</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="stockNameLabel">
       <property name="text">
        <string>Stock Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="stockNameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="stockPriceLabel">
       <property name="text">
        <string>Stock Price</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="stockPriceLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="stockNumberLabel">
       <property name="text">
        <string>Stock Number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="stockNumberLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addStockButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Stock</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
